x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/global_alerts_page.ts

```diff
@@ -102,10 +94,6 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         );
       });
 
-      afterEach(async () => {
```

Comment:
guess we don't need to cleanup the saved objects?

Reply:
Since we're not registering anything in the `objectRemover` in the first place I thought it wasn't necessary to do this (also, these tests are adaptations of the old internal alerts table tests so it might just be a leftover), wdyt?

Reply:
yea if theres nothing to clean then its all good :+1: